lang/python/python-paho-mqtt/Makefile

```diff
@@ -0,0 +1,58 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-paho-mqtt
+PKG_VERSION:=1.3.1
+PKG_RELEASE:=1
+PKG_MAINTAINER:=nobody
+PKG_LICENSE:=Eclipse Public License v1.0 / Eclipse Distribution License v1.0
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/eclipse/paho.mqtt.python
+PKG_SOURCE_VERSION:=v1.3.1
+
+PKG_SOURCE_SUBDIR:=$(BUILD_VARIANT)-paho-mqtt-$(PKG_VERSION)
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(BUILD_VARIANT)-paho-mqtt-$(PKG_VERSION)
+
+include $(INCLUDE_DIR)/package.mk
+include ../python-package.mk
+include ../python3-package.mk
+
+define Package/python-paho-mqtt/Default
+  SUBMENU:=Python
+  SECTION:=lang
+  CATEGORY:=Languages
+  MAINTAINER:=PierreF (https://github.com/PierreF)
```

Comment:
@thess correct me if I'm wrong but email is still preferred here (perhaps with GitHub id too)

Reply:
I have no idea what his email address is.

Reply:
Ah hah! I found it in the git log. Pierre Fersing <pierre.fersing@bleemeo.com>

Reply:
(From the [Contributing Guidelines](
https://github.com/openwrt/packages/blob/master/CONTRIBUTING.md):)

This should be the maintainer of the OpenWrt package (i.e. you), not the maintainer of the source software.

Also, a blank maintainer is allowed, though the package may not be reviewed as quickly. (Consider if you will have the time and energy to maintain this package in the future.)

Reply:
Hmm.  So why is there both a `PKG_MAINTAINER` at the top level and `MAINTAINER` in the `Package` definition?  I had presumed that `MAINTAINER` was for the software its self.  Is one of them deprecated?